### ISSUE TITLE
Set up blog with Ghost

### ISSUE THREAD

**Issue by jtremback** [althea-net/issue-warehouse#29]
- Talk to Justin about whether it should be self hosted or not.
- Get archive and blog template pages over

**Comment by asoltys** [comment]
Progress update: 

- Forked the default Ghost theme to bring in CSS, font, and layouts from https://althea.org: https://github.com/althea-mesh/ghost-theme
- Wrote a script to migrate blog posts from Jekyll by converting them into the JSON format expected by the Ghost import function: https://github.com/asoltys/jekyll-to-ghost-migration-script

Todo:

- Get blog images working properly
- Import posts from Medium (Jehan to provide data, Adam to do import)
- Setup staging site on VPS (Adam to do)
- Get hosted production account and setup domain blog.althea.org (Jehan and Justin to do)
- Get Theme and blog posts data imported into production site (Adam)

**Comment by jtremback** [comment]
> Import posts from Medium (Jehan to provide data, Adam to do import)

The data would be from our medium account

**Comment by asoltys** [comment]
Update:

Images are displayed now (https://github.com/althea-mesh/ghost-theme/commit/ca8aea436ba77ae46fde7de40e16e02a54696201)

There's no way to set an image caption for the featured image. We might be able to work around this by creating an image within the post which does allow you to add a caption and then tagging it with a special classname or something and then using Javascript to make it behave like the featured image. Any thoughts on that before I go ahead with it?

Medium only seems to let you export your own posts so we're going to need all the authors from Medium (Jehan, Justin, Deborah, Neeraj) to export their data separately. I can work on combining it all.

Staging site is here: https://altheamesh.com/toast/
Why /toast? /blog is currently redirecting to the Jekyll pages and ghost would let me use /ghost because it's used for admin pages. I'll come up with something better later but not sure how important this staging site is.

Production site is live: https://blog.althea.org/
Admin URL: https://blog.althea.org/ghost

The old Jekyll posts have been imported but some of the code snippets didn't get formatted properly and some image URLs are broken so I need to go through all the articles and check/fix them.

**Comment by jtremback** [comment]
I requested a download of our whole publication from medium support

**Comment by asoltys** [comment]
Medium posts have been imported. I noticed some images are spilling out of the container margins so I need to put some styles in to prevent that and I still need go through more posts to look for other issues.

**Comment by asoltys** [comment]
Ok, I think we can call this done now. I've gone through all the posts and fixed issues with broken links, code formatting, video embeds, missing tags, wrong authors, images spilling over, etc. I stripped off the unique ID's from the URL's of medium posts and updated all the internal links. Also went through the new static site and fixed a bunch of links there as well, including the main blog links in the top navigation and footer. Also added a javascript redirect on all the Jekyll posts being served from althea.org/blog/*

Oh and I added pagination controls to the index page. It displays 20 posts at a time.
